Adding cecret measles wrapper

erinyoung opened the pull request:
I've copied this file from dragen scripts. It copies measles wastewater files to /Volumes/NGS_2/measles and runs the Cecret workflow on the samples.

erinyoung committed:
Add files via upload

erinyoung committed:
Document run_cecret_measles_2.sh script usage

Added section for run_cecret_measles_2.sh script in README.

erinyoung commented on run_cecret_measles_2.sh:
```diff
@@ -0,0 +1,119 @@
+--primer_bed /Volumes/NGS_2/measles/references/NC_001498.1_primal_tiling_primer.bed \
+--species other --kraken2_db /Volumes/NGS_2/Data/kraken2_db/h+v/ \
+--samtools_ampliconstats false --nextclade_dataset nextstrain/measles/N450/WHO-2012 \
+--outdir $out_dir --freyja false
```
I think you are running the nextflow workflow twice. Can I adjust these lines to the following?

```
nextflow run UPHL-BioNGS/Cecret \
-profile uphl,slurm,mev \
--sample_sheet /Volumes/NGS_2/measles/$run_name/Cecret/${run_name}_samplesheet_cecret.csv \
--reference_genome /Volumes/NGS_2/measles/references/NC_001498.1.fasta \
--primer_bed /Volumes/NGS_2/measles/references/NC_001498.1_primal_tiling_primer.bed  \
--samtools_ampliconstats false \
--outdir $out_dir
```

tives82 commented:
I can't take credit for writing that wrapper. Pooja wrote that script. But
yeah, adjust the nextflow command as you suggested.

On Wed, Nov 5, 2025 at 12:25 PM Young ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In run_cecret_measles_2.sh
> <https://github.com/UPHL-BioNGS/General_LW_scripts/pull/57#discussion_r2495831399>
> :
>
> > +nextflow run UPHL-BioNGS/Cecret -profile docker \
> +--sample_sheet /Volumes/NGS_2/measles/$run_name/Cecret/${run_name}_samplesheet_cecret.csv \
> +--reference_genome /Volumes/NGS_2/measles/references/NC_001498.1.fasta \
> +--primer_bed /Volumes/NGS_2/measles/references/NC_001498.1_primal_tiling_primer.bed \
> +--species other --kraken2_db /Volumes/NGS_2/Data/kraken2_db/h+v/ \
> +--samtools_ampliconstats false \
> +--nextclade_dataset nextstrain/measles/N450/WHO-2012 \
> +--outdir $out_dir --freyja false
> +
> +# NEW way
> +
> +nextflow run /Volumes/NGS_2/Bioinformatics/eriny/Cecret -profile docker -resume \
> +--sample_sheet /Volumes/NGS_2/measles/$run_name/Cecret/${run_name}_samplesheet_cecret.csv \
> +--reference_genome /Volumes/NGS_2/measles/references/NC_001498.1.fasta \
> +--primer_bed /Volumes/NGS_2/measles/references/NC_001498.1_primal_tiling_primer.bed \
> +--species other --kraken2_db /Volumes/NGS_2/Data/kraken2_db/h+v/ \
> +--samtools_ampliconstats false --nextclade_dataset nextstrain/measles/N450/WHO-2012 \
> +--outdir $out_dir --freyja false
>
> I think you are running the nextflow workflow twice. Can I adjust these
> lines to the following?
>
> nextflow run UPHL-BioNGS/Cecret \
> -profile uphl,slurm,mev \
> --sample_sheet /Volumes/NGS_2/measles/$run_name/Cecret/${run_name}_samplesheet_cecret.csv \
> --reference_genome /Volumes/NGS_2/measles/references/NC_001498.1.fasta \
> --primer_bed /Volumes/NGS_2/measles/references/NC_001498.1_primal_tiling_primer.bed  \
> --samtools_ampliconstats false \
> --outdir $out_dir
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/UPHL-BioNGS/General_LW_scripts/pull/57#pullrequestreview-3423937425>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVOTXC7QERUVMRYUXYYENZL33JFLXAVCNFSM6AAAAACLH6TN36VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTIMRTHEZTONBSGU>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***>
>


-- 
Thomas Iverson, MS
Bioinformatician

Utah Public Health Laboratory

4431 South 2700 West

Taylorsville, UT 84129-8600

Main number: (801) 965-2400
***@***.***


*************** IMPORTANT MESSAGE ***************
This message, including any attachments, may contain confidential
information intended for a specific individual and purpose, and is
protected by law.  If you are not the intended recipient, delete this
message, including from trash, and notify me by telephone or email.

If you are not the intended recipient, any distributions or copying of this
message, or the taking of any action based on its content is strictly
prohibited.

-- 
*************** IMPORTANT MESSAGE ***************
This message, including 
any attachments, may contain confidential information intended for a 
specific individual and purpose, and is protected by law.  If you are not 
the intended recipient, delete this message, including from trash, and 
notify me by telephone or email.


If you are not the intended recipient, 
any distributions or copying of this message, or the taking of any action 
based on its content is strictly prohibited.

erinyoung committed:
Refactor Cecret Nextflow command to use local version

Updated the Nextflow command to run Cecret with the local version and removed the old command.

erinyoung commented:
@tives82 , I've made the change. Can you try it on the next run with measles wastewater? Or maybe an old run?

To get a specific branch

```
git clone -b adding-cecret-measles-wrapper https://github.com/UPHL-BioNGS/General_LW_scripts.git
```

tives82 commented:
I tried it on the last run and it worked fine.

tives82 reviewed (approved):
The cecret measles wrapper has been tested and works as expected.
